src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -815,33 +885,36 @@
   [jruby-service :- (schema/protocol jruby-protocol/JRubyPuppetService)
    get-code-content-fn :- IFn
    current-code-id-fn :- IFn
-   environment-class-cache-enabled :- schema/Bool]
-  (let [environment-class-handler
-        (environment-class-handler jruby-service
-                                   environment-class-cache-enabled),
+   cache-enabled :- schema/Bool]
+  (let [class-handler (environment-class-handler jruby-service
+                                                 cache-enabled)
+
+        module-handler (environment-module-handler jruby-service)
 
-        environment-module-handler
-        (environment-module-handler jruby-service)
+        tasks-handler (all-tasks-handler jruby-service)
 
-        all-tasks-handler
-        (all-tasks-handler jruby-service)
+        transport-handler (environment-transport-handler jruby-service
+                                                         cache-enabled)
 
-        task-details-handler
-        (task-details-handler jruby-service get-code-content-fn current-code-id-fn)
+        task-handler (task-details-handler jruby-service
```

Comment:
Ugh, I accidentally rebased away a maint commit where I changed the names of these handlers. Having them be the same name as fns w/in the namespace really bothered me. I can break this back out.